static/app/views/organizationStats/usageChart/index.tsx

```diff
@@ -43,36 +43,36 @@ export type CategoryOption = {
    * Scale of y-axis with no usage data.
    */
   yAxisMinInterval: number;
-} & SelectValue<DataCategory>;
+} & SelectValue<DataCategoryInfo['plural']>;
 
 export const CHART_OPTIONS_DATACATEGORY: CategoryOption[] = [
   {
-    label: DATA_CATEGORY_NAMES[DataCategory.ERRORS],
-    value: DataCategory.ERRORS,
+    label: DATA_CATEGORY_INFO.error.titleName,
+    value: DATA_CATEGORY_INFO.error.plural,
```

Comment:
Unfortunately, since the data category was being used as a plural in the URL bar, we can't force these to be the non-pluralized versions, since users may have bookmarked links. We can using the singular, backend names in the future, but this PR is meant to be non-destructive and purely semantic.